test/grid_animation.jl

```diff
@@ -0,0 +1,158 @@
+using Javis
+using Luxor
+
+function grid_animation()
+
+    video = Video(500, 500)
+    javis(
+        video,
+        [
+            Action(1:100, ground),
+            Action(1:100, :line, (args...) -> draw_grid(args..., grid_direction = :TL)),
+            Action(1:100, :line, (args...) -> zero_lines(args..., line_direction = :TL)),
+        ],
+        tempdirectory = "/home/src/Projects/javis/test/grid_test",
+        creategif = true,
+        pathname = "/home/src/Projects/javis/test/grid_test.gif",
+    )
+
+end
+
+function draw_grid(video::Video, action::Action, frame::Int; grid_direction::Symbol = :TR)
```

Comment:
For porting it into Javis itself you might want to use `Val` for dispatch here instead of having an `if` `else` for grid_direction. Just for an easier read (at least for me)